test/fixtures/EditionsDBService.scala

```diff
@@ -24,5 +24,11 @@ trait EditionsDBService extends DockerTestKit with DockerKitDockerJava with Dock
     editionsDB = new EditionsDB(dbUrl, dbUser, dbPassword)
   }
 
-  def withEvolutions[T]: (=> T) => T = Evolutions.withEvolutions[T](database)
+  before {
+    Evolutions.applyEvolutions(database)
+  }
+
+  after {
+    Evolutions.cleanupEvolutions(database)
+  }
```

Comment:
`withEvolutions` seems to take care of cleanup -- is this just to remove the need to nest test code in specs?

Reply:
This was a suggestion from @akash1810 and I also spent ages trying to figure out why my test failed (I'd forgotten to put it in `withEvolutions`) so I moved from `withEvolutions` to this. Pro: can't forget evolutions; Con: can't not do evolutions if you want to mix DB and non-DB tests.

Reply:
should these be `beforeEach` `afterEach`?

Reply:
`before` and `after` do happen on eash test. As opposed to `beforeAll` and `afterAll` that happen per suite. See http://doc.scalatest.org/1.8/org/scalatest/BeforeAndAfter.html